CIP-0072/version_1_offchain.json

```diff
@@ -11,63 +16,92 @@
       },
       "projectName": {
         "type":"string",
-        "description": "A project name, e.g. My dApp."
+        "description": "A project name, e.g. My dApp.",
+        "maxLength": 40
       },
       "link": {
         "type":"string",
         "description": "Website presenting a dApp.",
-        "pattern": "(https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|www\\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9]+\\.[^\\s]{2,}|www\\.[a-zA-Z0-9]+\\.[^\\s]{2,})"
-      },      
+        "pattern": "(https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|www\\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9]+\\.[^\\s]{2,}|www\\.[a-zA-Z0-9]+\\.[^\\s]{2,})",
```

Comment:
`www2.dapp.store` will not work. Suggest for all the url regex to be more permisive by removing the `www` check: https://mathiasbynens.be/demo/url-regex